android/app/src/main/java/org/mozilla/magnet/database/History.java

```diff
@@ -0,0 +1,75 @@
+package org.mozilla.magnet.database;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+public class History {
+    private static final String TAG = History.class.getName();
+    private static final long RECENT_TIME_PERIOD = TimeUnit.HOURS.toMillis(1);
+    private HistoryDatabase mDB;
+
+    /**
+     * Factory to create new `History` interface.
+     *
+     * @param context
+     * @return History
+     */
+    public static History create(Context context) {
+        return new History(new HistoryDatabaseSQL(context));
```

Comment:
We could do this a singleton, but don't see any other objects using this class so far, so we can leave it.